### DISCUSSION TITLE
EFCoreStore

### DISCUSSION THREAD

**Discussion by wmembreno** [Finbuckle/Finbuckle.MultiTenant#594] [General]
Good night, sorry if this sounds like a silly question, but I don't see anymore the option to use .WithEFCoreStore while using a .Net Core 6 project, is this a change for this version, or am I doing something wrong? I noticed the example available is for .Net Core 3.1, is out there any documentation or example to implement an EF Core Store on .Net Core 6? Thanks in advance for the help.

William.

**Comment by wmembreno** [comment]
Just to add some more, MultiTenantIdentityDbContext is not found in Finbuckle.MultiTenant, please help.

**Reply by AndrewTriesToCode** [reply]
Hi, what version number are you using? Can you try an earlier version to see if the problem is still there?

**Reply by wmembreno** [reply]
Hello Andrew, Im using .Net Core 6 as the project was developed on it, I was just adding multitenant capabilities to it. Will create an empty project for .net core 3.1 to test, will post some updates soon.

**Reply by wmembreno** [reply]
ok, what I did to fix was uninstall finbuckle  6.8.1, then installed 6.7.0, after that updated to 6.8.1 and then all is working, I can see both .WithEFCoreStore and I can derivate from MultiTenantIdentityDbContext.

**Reply by AndrewTriesToCode** [reply]
Ok thanks for the update and glad it was working. Please let me know if it happens again!
